internal/cli/kraft/cloud/deploy/deployer_kraftfile_runtime.go

```diff
@@ -105,6 +105,14 @@ func (deployer *deployerKraftfileRuntime) Deploy(ctx context.Context, opts *Depl
 		)
 	}
 
+	if opts.Project != nil && opts.Project.Env() != nil {
+		var projectEnv []string
+		for k, v := range opts.Project.Env() {
+			projectEnv = append(projectEnv, fmt.Sprintf("%s=%s", k, v))
+		}
+		opts.Env = append(projectEnv, opts.Env...)
```

Comment:
My processing power tells me this should be the other way around? If you set something from the CLI it should take precedence over the kraftfile